mobify/platform-scaffold
# AMP-42 - integration with the new deployment scripts in amp-sdk

**JIRA**: https://mobify.atlassian.net/browse/AMP-42
 **Linked PRs**: https://github.com/mobify/mobify-amp-sdk/pull/47

## Changes
- Cloudformation configs now live in Cloud. These shouldn't be exposed to the end user.
- Existing deploy scripts have been re-written in Python as server-side tasks.

## How to test-drive this PR
- Either wait until the linked PR is released or install a local version of the SDK, as usual.
- Run `npm run bundles -- --help` and check it makes sense.
- Run `npm run bundles -- list` - does it make sense?
- Run `npm run bundles -- push [path-to-bundle] --message "[some message here]"`
- Run `npm run bundles -- list` again and ensure the new bundle appears in the list.
- Run `npm run bundles -- publish [environment] [uuid]` and check the deployment works.
